src/wasm-ir-builder.h

```diff
@@ -310,12 +349,24 @@ class IRBuilder : public UnifiedExpressionVisitor<IRBuilder, Result<>> {
     return scopeStack.back();
   }
 
+  Result<ScopeCtx*> getScope(Index label) {
+    Index numLabels = scopeStack.size();
+    if (!scopeStack.empty() && scopeStack[0].isNone()) {
```

Comment:
What are the cases when a ScopeCtx has a type of None?

Reply:
This would be the case when parsing a global initializer or any other code that does not appear inside a function.